docs/contributing/coding-style.md

```diff
@@ -297,6 +297,51 @@ All errors should define in [internal/errors package](https://github.com/vdaas/v
 
 Please use [internal/errgroup](https://github.com/vdaas/vald/blob/master/internal/errgroup) for synchronized error handling on multi-goroutine processing.
 
+### Error checking
+
+All functions return `error` if the function can fail. It is very important to ensure the error checking is performed.
+To reduce human mistake that missing the error checking, please check the error using the following style:
+
+```golang
+// good
+if err := fn(); err != nil {
+    // handle error
+}
+```
+
+Instead of this style.
+
+```golang
+// bad
+err := fn()
```

Comment:
I can understand your opinion.
but It is also good to write that the following is correct when the function name is long.

```go
// good
err := thisHasAReallyLongSuperDescriptiveNameForSomeCrazyReason()
if err != nil {

}

```

Reply:
instead of this strategy, I think we should suggest NOT to declare long function name. What do you think?

Reply:
ok, it seems good to me.

Reply:
@vankichi what do you think?
If you support my idea, should I also update the `Methods` section.

Reply:
I think too long term for any methods make us confused in some cases.
we should use short but suitable terms for method, and we should cover using comment if it may occur some misreading.

Reply:
@vankichi I will update the `Method` section later